doc/physics_manual/sixphys.tex

```diff
@@ -977,48 +977,141 @@ \subsection{AC-dipole}
 
 \subsection{Electron lens}
 \label{elense}
-\subsubsection{Hollow electron lens - uniform annular profile}
-For a uniform distribution of the electron beam between $R_1$ and $R_2$, the radial kick can be described by a shape function $f(r)$ and a maximum kick strength $\theta_{\rm max}$:
+Electron lenses are devices where the tracked beam interacts with a beam of electrons. The electron beam travels co--axial to the main beam; it
+is contained in a thick cylinder of inner radius $R_1$ and outer radius $R_2$.
+If $R_1>0$, then the lens is hollow. The interaction is purely via the Lorentz
+force. These devices can be used for compensating beam--beam effects or as
+collimation devices~\cite{Shiltsev:2112966}.
+
+The implementation in SixTrack does not start from the Hamiltonian but from the
+integration of the Lorentz force along the length of the e--lens in idealised
+conditions. The e--lens \emph{must be a thin lens};
```

Comment:
Hyphenations should just use a single '-'. '--' is for dashes, like in beam–beam.

Reply:
that's exactly the case, i.e. elens is not a word :)